modules/ingest-geoip/src/internalClusterTest/java/org/elasticsearch/ingest/geoip/GeoIpProcessorNonIngestNodeIT.java

```diff
@@ -29,6 +30,7 @@
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 import static org.hamcrest.Matchers.equalTo;
 
+@AwaitsFix(bugUrl = "")
```

Comment:
kover somehow triggers running integTests too.
How can I limit kover to unit tests only?